repoclosure/yum.conf

```diff
@@ -17,34 +17,27 @@ module_platform_id=platform:el8
 [el8-baseos]
```

Comment:
Should we also update the labels to `stream8-baseos` and likewise for other repositories? I dont know if we will also need to update it somewhere else to not break things.

Reply:
I wouldn't do so. If you do, start with it in nightly and then see if it works.

Reply:
We probably want to wait for 3.1 to pass after we merge, after this one runs and get green, we can/should change Develop and 3.2, tagging @ekohl to see what he thinks

Reply:
I think I'd stick to el8. It's good enough here.